config/foreman.migrations/20210819164456_reset_puma_max_threads.rb

```diff
@@ -0,0 +1,9 @@
+if answers['foreman'].is_a?(Hash)
+  if answers['foreman']['foreman_service_puma_threads_max'] == 16
+    answers['foreman'].delete('foreman_service_puma_threads_max')
+
+    if answers['foreman']['foreman_service_puma_threads_min'] == 16
```

Comment:
Why 16? I don't think we ever had that as default. I think it should be 0. I'm also inclined to keep it separate from the `threads_max` condition.